joplin/api/schema.py

```diff
@@ -193,6 +194,117 @@ class Meta:
         model = LocationPageRelatedServices
         interfaces = [graphene.Node]
 
+class EventPageRemoteLocation(graphene.ObjectType):
+    value = GenericScalar()
+
+    street = graphene.String()
+    unit = graphene.String()
+    city = graphene.String()
+    state = graphene.String()
+    country = graphene.String()
+    zip = graphene.String()
+
+    def resolve_street(self, info):
+        return self.value['street']
+
+    def resolve_unit(self, info):
+        return self.value['unit']
+
+    def resolve_city(self, info):
+        return self.value['city']
+
+    def resolve_state(self, info):
+        return self.value['state']
+
+    def resolve_country(self, info):
+        return self.value['country']
+
+    def resolve_zip(self, info):
+        return self.value['zip']
+
+    name = graphene.String()
+
+    def resolve_name(self, info):
+        # We're doing our own translations in our model here
+        # so let's make sure the API still works as expected
```

Comment:
would be great to clarify for future readers what 'as expected' means

Reply:
addressed in new comments